internal/config/version.go

```diff
@@ -0,0 +1,3 @@
+package config
+
+var Version = "0.11.1" // x-release-please-version
```

Comment:
just for the sake of it (and maybe because it's less prone to conflicts) i would put this into a separate package

see https://github.com/canonical/identity-platform-login-ui/pull/177/commits/76b51edd463f5bed5e87d8b3daf6eb248c1a1d40 and https://github.com/canonical/identity-platform-login-ui/pull/177/commits/2362b4164ab5974fc3c652efae7f71d466aac690

you might be able to `cherrypick` them

Reply:
Sure, it makes sense. Should be fine now.